osu.Game/Users/UpdateableAvatar.cs

```diff
@@ -44,7 +44,7 @@ private void updateAvatar()
                 new Avatar(user)
                 {
                     RelativeSizeAxes = Axes.Both,
-                    OnLoadComplete = d => d.FadeInFromZero(200),
+                    OnLoadComplete = d => d.FadeInFromZero(400, Easing.Out),

```

Comment:
Considering this affects every avatar everywhere, 400 is probably a bit slow. Try 300 with `OutQuint`?